roles/matrix-jitsi/tasks/util/setup_jitsi_auth.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+#
+# Tasks related to setting up Jitsi authentication mechanisms
+#
+
+- name: Ensure matrix-jitsi-prosody container is running
+  systemd:
+    state: started
+    name: matrix-jitsi-prosody
```

Comment:
You can use `register` here to see if the "start" call actually started the service (`register: matrix_jitsi_prosody_start_result`).

Below, where you stop it, you can should avoid stopping if the start call did not do anything (indicating that `matrix-jitsi-prosody` had already been started anyway).

Reply:
The goal is to avoid stopping `matrix-jitsi-prosody` in case Jitsi is already running.

If you're handling a new installation (where none of the services are running yet), you can safely start `matrix-jitsi-prosody` and stop it later. No harm done.

However, if the user had started services (`--tags=start`), then starting `matrix-jitsi-prosody` will not do anything (because it's already started), but then stopping it will definitely stop it. And now all other services remain running, but would've effectively stopped `matrix-jitsi-prosody`. Not what the user expects.

Reply:
Ah, I didn't think of that. Thanks again :)
I'll fix this and submit right away.